sdks/common/fn-api/src/main/proto/beam_fn_api.proto

```diff
@@ -278,33 +154,20 @@ message ProcessBundleDescriptor {
   // refer to this.
   string id = 1;
 
-  // (Deprecated) A list of primitive transforms that should
-  // be used to construct the bundle processing graph.
-  //
-  // Migrate to Runner API definitions found within transforms field.
-  repeated PrimitiveTransform primitive_transform = 2 [deprecated = true];
-
-  // (Deprecated) The set of all coders referenced in this bundle.
-  //
-  // Migrate to Runner API defintions found within codersyyy field.
-  repeated Coder coders = 4 [deprecated = true];
-
   // (Required) A map from pipeline-scoped id to PTransform.
-  map<string, org.apache.beam.runner_api.v1.PTransform> transforms = 5;
+  map<string, org.apache.beam.runner_api.v1.PTransform> transforms = 2;
```

Comment:
Let's not be too mean to `MessageWithComponents`. It is easy to imagine other runners wanting to package up a tiny piece of a pipeline. We have far greater hacks in our codebase - this one you could consider a general utility :-)